spring-boot/proxy/src/main/java/eu/coatrack/proxy/metrics/MetricsCounterService.java

```diff
@@ -50,12 +52,18 @@ public class MetricsCounterService {
     public MetricsCounterService(MeterRegistry meterRegistry, MetricsTransmitter metricsTransmitter) {
         this.meterRegistry = meterRegistry;
         this.metricsTransmitter = metricsTransmitter;
+        // Quick fix of #237 by setting default time zones of Admin and Gateway to CET timezone.
+        // In the future, this system should be overhauled by the introduction of ZonedDateTime's.
+        TimeZone.setDefault(TimeZone.getTimeZone(ZoneId.of("Europe/Berlin")));
```

Comment:
If I understand [this article correctly](https://www.baeldung.com/java-jvm-time-zone), this command sets the timezone for the whole VM, not only for this class. Therefore it would be better to put it into the application's main class.

Reply:
When we change the metrics logic (e.g. by replacing it with Micrometer) then this class is removed and `TimeZone.setDefault(...)` is not required anymore. Since this command and the `MetricsCounterService` class belong together, I'd prefer to keep them in the same class.